src/molecules/rich-text/rich-text.stories.tsx

```diff
@@ -1,124 +0,0 @@
-/* eslint-disable react/no-danger */
```

Comment:
I think we also need a storybook file for TipTap editor? We'd also need to deploy the updated storybook later

Reply:
Restored and fit stories for rich text. Fixed stories for currency input.